style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: gold;
+}
```

Comment:
[OPTIONAL] Using color names is considered bad practice. I recommend you change this color to its `rgb` or `hex` equivalent.

Reply:
@29td Don't forget to request a second review through your dashboard.